include/RAJA/policy/cuda/teams.hpp

```diff
@@ -31,132 +31,8 @@ namespace RAJA
 namespace expt
 {
 
-template <typename BODY>
-__global__ void launch_global_fcn(LaunchContext ctx, BODY body_in)
-{
-  using RAJA::internal::thread_privatize;
-  auto privatizer = thread_privatize(body_in);
-  auto& body = privatizer.get_priv();
-  body(ctx);
-}
-
-template <bool async>
-struct LaunchExecute<RAJA::expt::cuda_launch_t<async, 0>> {
```

Comment:
Do we want to keep a specialization that does not use launch bounds? This would affect whether the launch policies default values for num_threads and BLOCKS_PER_SM should be 0 or 1.

Reply:
I wasn't sure whether to keep this, but didn't want to alter @artv3's interface, and left him the choice to decide/change this interface later. Yes, having `num_threads = 0` here makes things somewhat inconsistent.

Reply:
We should probably keep it then and let @artv3 remove/change it later. Its probably easier to let BLOCKS_PER_SM default to 1 then so it will always be valid.

Reply:
Sounds good, I'll change `BLOCKS_PER_SM` to default to 1 everywhere else.

Reply:
Turns out I did remove this already, as indicated by the red removed background . . . However, the original `cuda_launch_t` policy is intact for use in our testing framework.

Reply:
I mean that you should probably put this one back in, unremove it.

Reply:
Added this back as you suggested. Also changed the default `num_threads` to 1 because that is more consistent with other policies, and the number of threads is not actually used in this policy anyway. Documented this in the code as well. Hopefully this is ok with @artv3?

Reply:
Enhancements always welcomed! 👍